app/models/message_template.rb

```diff
@@ -0,0 +1,9 @@
+class MessageTemplate < ApplicationRecord
```

Comment:
Ich bin nicht ganz sicher, ob der in der Tech-Spec vorgeschlagene Name wirklich passend ist. Soll das Template allgemein verwendet werden oder nur für die Rechnungen? Falls 2.) würde ich vorschlagen, das noch auf `InvoiceTextTemplate` o.Ä. umzubenennen.